features/steps/setupSelenium.py

```diff
@@ -3,5 +3,5 @@
 options = Options()
 options.headless = True
 driver = webdriver.Chrome(options=options)
-website_url = "http://localhost:8000/"
```

Comment:
It will fail if you don't run the web app on port 5000 but by default, the web app runs on port 5000. We run the web app it on port 8000 manually because the latest versions of MacOS use port 5000 so the web app goes down sometimes. I don't know if this is related to your question  😄

Reply:
yep thats right the change was from an old version of the file pycharm kept smart checking out for me, this has been reverted